src/main/scala/org/broadinstitute/dsde/firecloud/model/ModelSchema.scala

```diff
@@ -59,3 +43,41 @@ case class EntityMetadata(
 )
 
 case class EntityModel(schema : Map[String, EntityMetadata]) //entity name -> stuff about it
+
```

Comment:
Maybe some code comments to explain FlexibleModelSchema and ModelSchema and how they relate?